ComponentKit/Components/CKButtonComponent.mm

```diff
@@ -95,6 +95,10 @@ + (instancetype)newWithTitles:(const std::unordered_map<UIControlState, NSString
                    attributes:(const CKViewComponentAttributeValueMap &)passedAttributes
    accessibilityConfiguration:(CKButtonComponentAccessibilityConfiguration)accessibilityConfiguration
 {
+  const auto titles = titlesParam.take();
```

Comment:
This operation moves the value out of the wrapped container in the same manner as `std::move` would.

A conversion that simply returns a const-reference makes it extremely easy to create an inadvertent copy. A conversion that actually moves the container is extremely confusing.